### PR TITLE
translate linux md file

### PR DESCRIPTION
**Pull request by sunnywa**
Signed-off-by: 王琦 <1099046071@qq.com>

### CODE REVIEW

**Comment by sunnywa** [comment]
@alabulei1

File: docs/book/zh/src/os/linux.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+| `ubuntu-build-clang`    | x86\_64 | Ubuntu 20.04 LTS       | 12.0.0       | CC=clang, CXX=clang++ | Ubuntu 20.04+            | 始终使用最新的 Ubuntu 版本提供给 CI                            |
+| `ubuntu2004_x86_64`     | x86\_64 | Ubuntu 20.04 LTS       | 10.0.0       | CC=gcc, CXX=g++       | Ubuntu 20.04+            | 提供给熟悉 Ubuntu 20.04 LTS 版本的开发者                  |
+| `ubuntu2104_armv7l`     | armhf   | Ubuntu 21.04           | 12.0.0       | CC=gcc, CXX=g++       | Ubuntu 21.04+            | 提供给 armhf 发布的版本                                                           |
+| `manylinux2014_x86_64`  | x86\_64 | CentOS 7, 7.9.2009     | 12.0.0       | CC=gcc, CXX=g++       | Ubuntu 16.04+, CentOS 7+ | 提供给熟悉 X86_64 架构的 CentOS 的开发者           |
+| `manylinux2014_aarch64` | aarch64 | CentOS 7, 7.9.2009     | 12.0.0       | CC=gcc, CXX=g++       | Ubuntu 16.04+, CentOS 7+ | 提供给熟悉 Aarch64 架构的 CentOS 的开发者           |
+| `manylinux2010_x86_64`  | x86\_64 | CentOS 6, 6.10         | 12.0.0       | CC=gcc, CXX=g++       | Ubuntu 14.04+, CentOS 6+ | 提供给熟悉 x86_64 架构的遗留系统的开发人员准备的 |
```
Comment (alabulei1) [inline docs/book/zh/src/os/linux.md:15]: EOL  也加上吧

File: docs/book/zh/src/os/linux.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+| `manylinux1_x86_64`     | x86\_64 | CentOS 5, 5.11         | 12.0.0       | CC=gcc, CXX=g++       | Ubuntu 14.04+, CentOS 5+ | 提供给熟悉 x86_64 架构的遗留系统的开发人员准备的 |
```
Comment (alabulei1) [inline docs/book/zh/src/os/linux.md:16]: EOL 也加上吧

**Review by alabulei1 — COMMENTED** [review]
@sunnywa Thanks for your contribution. Please check out my comments.

**Comment by sunnywa** [comment]
@alabulei1 已完成

**Comment by alabulei1** [comment]
LGTM! Thanks @sunnywa 

@hydai  Could you help merge this PR?

**Comment by sunnywa** [comment]
@hydai Could you help merge this PR?

**Review by hydai — APPROVED** [review]

**Pull request merged**